js/examples/notebooks/deno_datasets_and_experiments_quickstart.ipynb

```diff
@@ -0,0 +1,443 @@
+   "source": [
+    "Set up your OpenAI API key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "const openaiApiKey = prompt(\"Enter your OpenAI API key:\");\n",
+    "\n",
+    "if (!openaiApiKey) {\n",
+    "  console.error('Please enter your OpenAI API key to continue');\n",
+    "  Deno.exit(1);\n",
+    "}\n",
+    "\n",
+    "const openai = new OpenAI({\n",
+    "  apiKey: openaiApiKey,\n",
+    "});"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Initialize the Phoenix client."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "const client = createClient();\n",
+    "console.log('Phoenix client initialized. Access Phoenix UI at http://localhost:6006');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating a Dataset\n",
+    "\n",
+    "Let's create examples for our dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "console.log('Creating dataset examples...');\n",
+    "\n",
+    "// Create examples directly as an array\n",
+    "const { datasetId } = await createDataset({\n",
+    "  name: `quickstart-dataset-${Date.now()}`,\n",
+    "  description: \"Dataset for quickstart example\",\n",
+    "  examples: [\n",
+    "  {\n",
+    "    id: `example-1`,\n",
+    "    updatedAt: new Date(),\n",
+    "    input: { question: \"What is Paul Graham known for?\" },\n",
+    "    output: { answer: \"Co-founding Y Combinator and writing on startups and techology.\" },\n",
+    "    metadata: { topic: \"tech\" }\n",
+    "  },\n",
+    "  {\n",
+    "    id: `example-2`,\n",
+    "    updatedAt: new Date(),\n",
+    "    input: { question: \"What companies did Elon Musk found?\" },\n",
+    "    output: { answer: \"Tesla, SpaceX, Neuralink, The Boring Company, and co-founded PayPal.\" },\n",
+    "    metadata: { topic: \"entrepreneurs\" }\n",
+    "  },\n",
+    "  {\n",
+    "    id: `example-3`,\n",
+    "    updatedAt: new Date(),\n",
+    "    input: { question: \"What is Moore's Law?\" },\n",
+    "    output: { answer: \"The observation that the number of transistors in a dense integrated circuit doubles about every two years.\" },\n",
+    "    metadata: { topic: \"computing\" }\n",
+    "  }\n",
+    "]})\n",
+    "\n",
+    "console.log(`examples created`);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Defining the Task\n",
+    "\n",
+    "Define the task function that will be evaluated."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "const taskPromptTemplate = \"Answer in a few words: {question}\";\n",
+    "\n",
+    "const task = async (example) => {\n",
```

Comment:
Can we annotate this with the Task function type so that you get type hinting on example and the return type?

```suggestion
    "import { type RunExperimentParams } from \"@arizeai/phoenix-client/experiments\";\n",
    "\n",
    "const task: RunExperimentParams["task"] = async (example) => {\n",
```